docs/how-to-guides/running-feast-in-production.md

```diff
@@ -77,16 +116,38 @@ In the above example we are materializing the source data from the `driver_hourl
 
 The timestamps above should match the interval of data that has been computed by the data transformation system.
 
-## 3. How to use Feast for model training and serving
+### 2.2. Automate periodic materializations 
+
+It is up to you which orchestration/scheduler to use to periodically run `$ feast materialize`.
+Feast keeps the history of materialization in its registry, so the choice could be as simple as a [unix cron util](https://en.wikipedia.org/wiki/Cron).
+However, with this option materialization work is not distributed, and you might end up limited by the resources of a single machine. 
+If you have just a few materialization jobs (usually it's one materialization job per feature view) and you don't run them very frequently, this is fine.
+Otherwise, you might want to use a job orchestrator to run multiple jobs in parallel using several workers.
+Kubernetes Jobs or Airflow are good choices for more comprehensive job orchestration.
+
+If you are using Airflow as a scheduler, Feast can be invoked through the [BashOperator](https://airflow.apache.org/docs/apache-airflow/stable/howto/operator/bash.html) after the [Python SDK](https://pypi.org/project/feast/) has been installed into a virtual environment and your feature repo has been synced:
+```python
+materialize = BashOperator(
+    task_id='materialize',
+    bash_command=f'feast materialize-incremental {datetime.datetime.now().replace(microsecond=0).isoformat()}',
+)
+```
+
+{% hint style="success" %}
+Important note: Airflow worker must have read and write permissions to the registry file on GS / S3
+since it pulls configuration and updates materialization history.
+{% endhint %}
+
+## 3. How to use Feast for model training
 
-Now that you have deployed a registry, provisioned your feature store, and loaded your data into your online store, your clients can start to consume features for training and inference.
+After we defined our features and data sources in the repository we can start working on training by generating a training dataset.
 
-For both model training and inferencing your clients will use the Feast Python SDK to retrieve features. In both cases it is necessary to create a `FeatureStore` object.
+The first thing we need to do in our training code is to create a `FeatureStore` object with a path to the registry.
```

Comment:
feature repository make more sense because registry may refer to registry.db
```suggestion
The first thing we need to do in our training code is to create a `FeatureStore` object with a path to the feature repository.
```

Reply:
That's exactly what was meant here. It's the path to registry.db.
Feature Repository on the other hand is where the user keeps her feature definitions.